integration-tests/src/test/java/org/zowe/apiml/functional/gateway/AuthenticationOnDeploymentTest.java

```diff
@@ -64,91 +60,6 @@ void setUp() {
         verifier.clear();
     }
 
```

Comment:
I agree that this test should be removed(fixed), but wouldn't be better to do it in separate PR?

Reply:
Valid point, returned the test.